.travis.yml

```diff
@@ -93,3 +91,23 @@ script:
         return 1 ;
         ;;
     esac
+
+
+after_success: |
+    case "$TEST_KIND" in
+      comprakt-release)
+        cargo doc \
```

Comment:
`--document-private-items`?

Otherwise r=me

Reply:
Yeah, I don't know, do we want that?

Reply:
otherwise documentation of every bin crate will be empty

Reply:
![image](https://user-images.githubusercontent.com/9744647/48859079-6b52f200-edbd-11e8-8b0e-d9e27046bb8a.png)

vs only documentation for `LoweringOptions`